client/src/App.js

```diff
@@ -33,14 +37,23 @@ const App = () => {
 		//  data: someData
 	}
 
-
+	const Routing = () => {
```

Comment:
Perhanps consider a name change for this component so we don't have three different components with such similar names. Otherwise, you can just put the  Routes component directly in App's return

Reply:
Also, if this Routes component handles exclusively popups, either make that clear with the name change, or just don't use routes for popups, opting instead to conditionally render them (look into bootstrap modals).